webapp/controller/ObjectPage.controller.js

```diff
@@ -0,0 +1,64 @@
+sap.ui.define([
+	"sap/ui/core/mvc/Controller",
+	"sap/ui/model/json/JSONModel",
+	"sap/base/strings/formatMessage"
```

Comment:
You should not define format message in controller file. You can import it within xml file. Please check in details the examples I've provided in notion